docs/GettingStarted.md

```diff
@@ -2637,15 +2636,12 @@ or `c.agent.port`.
 
 #### Transporting in Test Mode
 
-When test mode is enabled, the tracer uses a `Test` adapter for no-op transport that can optionally buffer requests in
-test suites or other non-production environments. It is configured by setting `c.tracing.test_mode.enabled` to true.
-This mode only works for tracing.
+The test mode can be used for testing distributed tracing behavior. Enabling this setting will mean every
+span will be sent (no sampling is applied) and the transport will send data synchronously.
+It is configured by setting `c.tracing.test_mode.enabled` to true.
 
-```ruby
-Datadog.configure do |c|
-  c.tracing.test_mode.enabled = true
-end
-```
+If you're looking to disable tracing while your test suite is running, consider using the `tracing.enabled` setting
+to disable sending of traces.
```

Comment:
A minor thing, but it would be cool to give an example on how-to, even if it's obvious

```suggestion
If you're looking to disable tracing while your test suite is running, consider using the `tracing.enabled` setting
to disable sending of traces.

```ruby
Datadog.configure do |c|
  c.tracing.test_mode.enabled = false
end
```
```

Reply:
Wait, I'm somewhat confused with this suggestion -- did you mean to suggest an example of using `tracing.enable`?

I explicitly removed the example with `test_mode.enabled` and stated why (item 4) so I'm not sure I understand your suggestion :eyes: